proposals/paymentapps/payment-apps.html

```diff
@@ -337,25 +337,42 @@
             process a payment request that conforms to the rules defined in the
             <a>payment method specification</a> for that payment
             method and SHOULD be able to return an appropriate response.</p>
-        <p>However, sometimes an app will be designed to support a specific
-            payment method but the app will not have been configured with the
-            appropriate <a>payment method data</a> (such as user credentials)
-            to process payments using that payment method. In that case the
-            <a>payment app</a> is said to <code>support</code> the payment
-            method but that payment method is not <code>enabled</code>.</p>
-        <p>For example, a <a>payment app</a> may be capable of processing a
-            credit card payment by simply returning the card details in the
-            payment response but unless the user has configured the payment app
-            with the card details or is able to provide these at the time of
-            processing that method is not <code>enabled</code>.</p>
-        <p>If a payment app defines a payment method as <code>enabled</code>
-            then it MUST be able to process any payment
-            request that lists that payment method as supported and is
-            formatted correctly according to the <a>payment method
-            specification</a> for that payment method.</p>
-        <p><a>User agents</a> should only consider <code>enabled</code>
-            payment methods when evaluating if a <a>payment app</a> is
-            appropriate to process a given payment request.</p>
+        <p>For a given <a>payment method</a> supported by a given <a>payment
+            app</a>, the <a>payment method</a> can be in one of two states:</p>
+        <dl>
+            <dt><dfn id="payment-app-enabled-payment-method">enabled</dfn></dt>
+            <dd>
+                <p>A <a>payment app</a> can define a payment method as <a>enabled
+                    </a> by adding it to the <code><a>enabled_methods</a></code>
+                    list. If a payment app defines a payment method as <a>enabled
+                    </a>, then it MUST be able to process any payment
+                    request that lists that payment method as supported and is
+                    formatted correctly according to the <a>payment method
+                    specification</a> for that payment method.</p>
+                <p>A <a>user agent</a> SHOULD NOT define a payment method as <a>
+                    enabled</a> unless one of the following criteria is met:
+                    </p>
```

Comment:
What does it mean for a **user agent** to define a payment method as enabled? The user agent doesn't know anything except what the payment app tells it during registration.